lib/scan/dir/engine.py

```diff
@@ -198,6 +188,10 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
             for idir in extra_requirements["dir_scan_list"]:
                 if random_agent_flag:
                     user_agent = {'User-agent': random.choice(user_agent_list)}
+                if target.endswith("/"):
+                    target = target[:-1]
```

Comment:
Just because tool should not request:
https://pintasuper.com**//**dir.
Instead should search:
https://pintatsuper.com**/**dir.